src/services/authService.ts

```diff
@@ -0,0 +1,27 @@
+import { createUserWithEmailAndPassword, AuthError } from 'firebase/auth';
+import { setDoc, doc } from 'firebase/firestore';
+import { auth, db } from '@/firebase/index';
```

Comment:
sin el index también apunta al mismo archivo
```suggestion
import { auth, db } from '@/firebase';
```